src/builder/Build.kt

```diff
@@ -65,7 +65,10 @@ class Build(val waves: Map<Int, Wave>, val global: Global) {
     }
 
     fun survivability(wave: Wave): String {
-        val calc = BattleCalc(global, lane.fighters, (0 until wave.amount).map { wave.creatures.first() })
+        val calc = BattleCalc(global, lane
+                .getFighters(currentLevel).values
+                .filter { it.unitClass != UnitClass.Worker }

```

Comment:
Maybe add this filter to getFighters? Later we could add a workercount function or do you think it makes sense to show the workers in the lane view?